parser/proto.y

```diff
@@ -327,269 +334,231 @@ stringLit : _STRING_LIT {
 		$$ = &stringList{$1, $2}
 	}
 
-aggregate : '{' aggFields '}' {
+messageLiteralWithBraces : '{' messageTextFormat '}' {
 		fields, delims := $2.toNodes()
 		$$ = ast.NewMessageLiteralNode($1, fields, delims, $3)
 	}
 	| '{' error '}' {
 	    $$ = nil
 	}
 
-aggFields : aggField {
-		if $1 != nil {
-			$$ = &messageFieldList{$1, nil}
-		} else {
-			$$ = nil
-		}
-	}
-	| aggField aggFields {
-		if $1 != nil {
-			$$ = &messageFieldList{$1, $2}
-		} else {
-			$$ = $2
-		}
-	}
-	| {
-		$$ = nil
+messageTextFormat : messageLiteralFields {
+		$$ = $1
 	}
 
```

Comment:
When the code block is an "identity transform" and both the source and target elements have the same type (which is true in this case, since these both use `%type <msgLitFlds>`), you can omit the block.
```suggestion
messageTextFormat : messageLiteralFields

```

Reply:
I fixed this and other "identity transform".